client/src/components/rooms/RoomsMeetingExperience.tsx

```diff
@@ -418,6 +419,22 @@ const RoomsMeetingExperience = (props: RoomsMeetingExperienceProps): JSX.Element
     );
   }
 
+  if (userRole === RoomParticipantRole.presenter && renderPostCall && transcriptionFeatureEnabled) {
```

Comment:
`renderPostCall` is confusing, what is rendered. Suggestion: `renderEndCallScreen`